libs/libwebsockets/Makefile

```diff
@@ -43,32 +43,37 @@ define Package/$(PKG_NAME)/Default
 	SECTION:=libs
 	CATEGORY:=Libraries
 	TITLE:=libwebsockets
-	DEPENDS:=+zlib
 	URL:=https://libwebsockets.org
 	MAINTAINER:=Karl Palsson <karlp@etactica.com>
```

Comment:
Sure, this was just the style in effect at the time, and I wasn't going around making changes for fun :)